README.md

```diff
@@ -76,9 +76,9 @@ Any new benchmarks must be named `*_benchmark.dart` and reside in the
 - [ ] along
 - [ ] area
 - [ ] bbox
```

Comment:
Ok. just found after merging, that bbox is not mentioned here

Reply:
done!